packages/calling/src/CallSettings/CallSettings.ts

```diff
@@ -17,9 +17,17 @@ import {CALL_SETTINGS_FILE} from './constants';
 import {UcmBackendConnector} from './UcmBackendConnector';
 
 /**
- * Call Settings class which implements the public ICallSettings interface
- * This class will select appropriate calling backends according to the user
- * entitlements.
+ * Purpose of the CallSettings Client instance is to provide the APIs to fetch and update the settings like CallWaiting, DND, CallForward, Voicemail etc. based on different calling backends.
+ * The appropriate calling backends is initialized according to the user entitlements while instantiating the CallSettings Client.
+ *
+ * To access the APIs, instance of CallSettings Client is required.
```

Comment:
Can remove this sentence from here as well as other places. The example is self-explanatory.

Reply:
Done